test/TestProjects/MgmtKeyvault/tests/Generated/Mock/VaultCollectionTest.cs

```diff
@@ -36,8 +36,47 @@ public async Task CreateOrUpdate()
                 EnabledForDiskEncryption = true,
                 EnabledForTemplateDeployment = true,
             });
+            vaultCreateOrUpdateParameters.Properties.AccessPolicies.Add(new MgmtKeyvault.Models.AccessPolicyEntry(tenantId: Guid.Parse("00000000-0000-0000-0000-000000000000"), objectId: "00000000-0000-0000-0000-000000000000", permissions: new MgmtKeyvault.Models.Permissions()));
+            vaultCreateOrUpdateParameters.Properties.AccessPolicies[0].Permissions.Keys.Add(new MgmtKeyvault.Models.KeyPermissions("encrypt"));
+            vaultCreateOrUpdateParameters.Properties.AccessPolicies[0].Permissions.Keys.Add(new MgmtKeyvault.Models.KeyPermissions("decrypt"));
+            vaultCreateOrUpdateParameters.Properties.AccessPolicies[0].Permissions.Keys.Add(new MgmtKeyvault.Models.KeyPermissions("wrapKey"));
```

Comment:
The object initialization here is not quite intuitive... but since it could work, we could refactor this later